src/hotspot/share/opto/loopnode.cpp

```diff
@@ -2988,11 +2989,75 @@ void OuterStripMinedLoopNode::fix_sunk_stores(CountedLoopEndNode* inner_cle, Loo
   }
 }
 
+// Sunk stores should be referenced from an outer loop memory Phi
+void OuterStripMinedLoopNode::handle_sunk_stores_at_expansion(PhaseIterGVN* igvn) {
```

Comment:
What does the word "expansion" refer to? Could you also mention that in your code comment above, please?

Reply:
I renamed that one.